src/display/api.js

```diff
@@ -1491,6 +1502,14 @@ var PDFWorker = (function PDFWorkerClosure() {
     return new PDFWorker(null, port);
   };
 
+  PDFWorker.getWorkerSrc = function() {
+    try {
```

Comment:
Why try-catch? `getWorkerSrc` is not expected to throw (unless there is no way to determine the worker source.

Reply:
While you're correct that it *shouldn't* throw, than may still happen in some cases [given this code](https://github.com/mozilla/pdf.js/blob/55e3f97aa97ad648ef955203524bbdb5593671d1/src/display/api.js#L1217)

In e.g. Node.js, where workers are disabled by default and `workerSrc` isn't set in the examples, if that function is called a (perhaps unexpected) error would be thrown. With the `getWorkerSrc` function exposed to the outside, I simply wanted to avoid "surprising" anyone using this function.

Reply:
While you're correct that it *shouldn't* throw, than may still happen in some cases [given this code](https://github.com/mozilla/pdf.js/blob/55e3f97aa97ad648ef955203524bbdb5593671d1/src/display/api.js#L1217).

In e.g. Node.js, where workers are disabled by default and `workerSrc` isn't set in the examples, if that function is called a (perhaps unexpected) error would be thrown.
With the `getWorkerSrc` function now exposed to the outside, it seemed nice to avoid "surprises" for an API consumer who uses that function.

Reply:
While you're correct that it *shouldn't* throw, that could still happen [given this code](https://github.com/mozilla/pdf.js/blob/55e3f97aa97ad648ef955203524bbdb5593671d1/src/display/api.js#L1217).
Hence it seemed nice to try and avoid surprises for an API consumer that uses this function not expecting an `Error` to be thrown. Are you saying that you think this is a completely unwarranted concern, and that should return the internal `getWorkerSrc` value as-is?

Reply:
Yes, I suggest to not catch the error. `workerSrc` doesn't make sense in Node.js, and when it does, we have to provide a suitable implementation instead of silently returning nothing.

Reply:
I've make the requested changes; thanks for the review comments!